app/views/candidate_interface/application_form/_review.html.erb

```diff
@@ -97,13 +97,23 @@
   )) %>
 
   <h3 class="govuk-heading-m"><%= t('page_titles.degree') %></h3>
-  <%= render(CandidateInterface::DegreesReviewComponent.new(
-    application_form: application_form,
-    editable: editable, heading_level: 4,
-    show_incomplete: true,
-    missing_error: missing_error,
-    return_to_application_review: true
-  )) %>
+  <% if FeatureFlag.active?(:new_degree_flow) %>
+    <%= render(CandidateInterface::DegreeNewReviewComponent.new(
```

Comment:
Just wondering if we're planning to rename these at some point, so there's no `new` in the name?

Reply:
ye it was just to differentiate them whilst the feature flag was active, when we can remove might as well rename?